src/Tribe/Properties/Last_Registered_Ticket.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Tribe\HubSpot\Properties;
+
+/**
+ * Class Last_Registered_Ticket
+ *
+ * @package Tribe\HubSpot\Properties
+ */
+class Last_Registered_Ticket extends Base {
+
+	/**
+	 * @var string
+	 */
+	protected $properties_grouping_name = 'last_registered_ticket';
+
+	/**
+	 * @var array
+	 */
+	protected $properties = [];
+
+	public function __construct() {
+
+		$this->set_properties();
+	}
+
+	/**
+	 * Set the Individual Properties for this Grouping
+	 *
+	 * @since 1.0
+	 *
+	 */
+	public function set_properties() {
+
+		$this->properties = [
+			'last_registered_ticket_type_id'       => [
+				'label'     => 'Last Registered Ticket ID',
```

Comment:
Should all labels use __() or are these the ones we send directly to HubSpot?

Reply:
I added translation ability as the id stays the same so it could be translated in HubSpot.